diff --git a/lzf/lzf_filter.c b/lzf/lzf_filter.c
index c6dd4b0..08579a7 100644
--- a/lzf/lzf_filter.c
+++ b/lzf/lzf_filter.c
@@ -26,7 +26,7 @@
 #include <stdio.h>
 #include <errno.h>
 #include "hdf5.h"
-#include "lzf/lzf.h"
+#include "lzf.h"
 #include "lzf_filter.h"
 
 /* Our own versions of H5Epush_sim, as it changed in 1.8 */
diff --git a/setup.py b/setup.py
index d92ee52..293b1ac 100755
--- a/setup.py
+++ b/setup.py
@@ -120,7 +120,7 @@ if sys.platform.startswith('win'):
         COMPILER_SETTINGS['library_dirs'] += [op.join(HDF5, 'dll')]
 else:
     COMPILER_SETTINGS = {
-       'libraries'      : ['hdf5', 'hdf5_hl'],
+       'libraries'      : ['hdf5', 'hdf5_hl', 'lzf'],
        'include_dirs'   : [numpy.get_include(), localpath('lzf')],
        'library_dirs'   : [],
        'define_macros'  : [('H5_USE_16_API', None)]
@@ -167,9 +167,8 @@ if HAVE_CYTHON:
 else:
     configure.printerr("Cython not present; building for HDF5 1.8.4+")
 
-EXTRA_SRC = {'h5z': [ localpath("lzf/lzf_filter.c"),
-                      localpath("lzf/lzf/lzf_c.c"),
-                      localpath("lzf/lzf/lzf_d.c")]}
+EXTRA_SRC = {'h5z': [ localpath("lzf/lzf_filter.c")]}
+
 
 def make_extension(module):
     sources = [op.join('h5py', module+SUFFIX)] + EXTRA_SRC.get(module, [])
